RecoLocalFastTime/FTLRecProducers/plugins/BuildFile.xml

```diff
@@ -7,6 +7,7 @@
 <use   name="FWCore/ParameterSet"/>
```

Comment:
this has a dependency on ```RecoLocalFastTime/FTLRecProducers```
which is technically empty now.
I'm not even sure if  RecoLocalFastTime/FTLRecProducers/BuildFile.xml is even needed.